pcdsdevices/spectrometer.py

```diff
@@ -521,3 +521,29 @@ class Gen1VonHamos4Crystal(BaseInterface, GroupDevice):
     cr2 = Cpt(Gen1VonHamosCrystal, '', trans_axis=':03', yaw_axis=':07', pitch_axis=':11' , kind='normal', name='Crystal 2')
     cr3 = Cpt(Gen1VonHamosCrystal, '', trans_axis=':04', yaw_axis=':08', pitch_axis=':12' , kind='normal', name='Crystal 3')
     cr4 = Cpt(Gen1VonHamosCrystal, '', trans_axis=':05', yaw_axis=':09', pitch_axis=':13' , kind='normal', name='Crystal 4')
+
+
+class DCCMono(BaseInterface, GroupDevice):
+    """
+    Double Channel Cut Monochrometer controlled with a Beckhoff PLC. 
+    This includes five axes in total: 
+	- 2 for crystal manipulation (TH1/Upstream and TH2/Downstream)
+	- 1 for chamber translation in x direction (TX)
+	- 2 for YAG diagnostics (TXD and TYD)
+
+    Parameters
+    ----------
+    prefix : str,optional
+	Base PV for DCCM motors	
+    name : str
+	A name or alias to refer to the device. 
+    """
+
+    tab_component_names = True
+    TH1 = Cpt(BeckhoffAxis, ':MMS:TH1', doc = 'Bragg Upstream/TH1 Axis', kind = 'normal')
+    TH2 = Cpt(BeckhoffAxis, ':MMS:TH2', doc = 'Bragg Downstream/TH2 Axis', kind = 'normal')
+    TX = Cpt(BeckhoffAxis, ':MMS:TX', doc = 'Translation X Axis', kind = 'normal')
+    TXD = Cpt(BeckhoffAxis, ':MMS:TXD', doc = 'YAG Diagnostic X Axis', kind = 'normal')
+    TYD = Cpt(BeckhoffAxis, ':MMS:TYD', doc = 'YAG Diagnostic Y Axis', kind = 'normal')
```

Comment:
The python pep8 naming standards (https://peps.python.org/pep-0008/) usually reserve capital letters for constants. It's convention to name class attributes like components/subdevices in all lowercase (with underscores if needed).
```suggestion
    th1 = Cpt(BeckhoffAxis, ':MMS:TH1', doc = 'Bragg Upstream/TH1 Axis', kind = 'normal')
    th2 = Cpt(BeckhoffAxis, ':MMS:TH2', doc = 'Bragg Downstream/TH2 Axis', kind = 'normal')
    tx = Cpt(BeckhoffAxis, ':MMS:TX', doc = 'Translation X Axis', kind = 'normal')
    txd = Cpt(BeckhoffAxis, ':MMS:TXD', doc = 'YAG Diagnostic X Axis', kind = 'normal')
    tyd = Cpt(BeckhoffAxis, ':MMS:TYD', doc = 'YAG Diagnostic Y Axis', kind = 'normal')
```